### DISCUSSION TITLE
How do I qualify for GitHub's Student Developer Pack, and what documentation should I provide?

### DISCUSSION THREAD

**Discussion by ft-dev53** [community/community#143729] [GitHub Education] [answered]
### Hi everyone,

Hi everyone,

I'm a student interested in applying for GitHub's Student Developer Pack to access resources and tools that would really help with my studies. However, I want to make sure I submit the correct information and qualify without any issues.

Could someone please clarify the eligibility requirements? Also, what type of documentation do I need to provide to ensure my application is accepted? Any tips or advice would be greatly appreciated!

Thanks in advance!

**Comment by BbySharp-dev** [comment, marked as answer]
The GitHub Student Developer Pack offers a fantastic set of resources for students, and it’s definitely worth applying. Here’s a breakdown of what you need to know:

Eligibility Requirements
To qualify for the GitHub Student Developer Pack, you must:

Be currently enrolled in a degree- or diploma-granting course, or be a student in a K-12 educational institution.
Be at least 13 years old (or the minimum age in your country to hold a GitHub account).
Use your GitHub account for non-commercial purposes during your studies.
Required Documentation
To prove your student status, you’ll need to provide one of the following documents:

School-Issued Student ID Card: Make sure your name, your photo, and your school’s name are visible. Avoid submitting an edited or cropped version, as GitHub’s verification system may flag altered images.
Enrollment Verification Letter: This is an official document from your institution confirming your student status. It should be current and show your enrollment details.
Transcript or Enrollment Certificate: This document should also display your name and the institution’s name.
Tips for a Successful Application
Submit Clear Documentation: Ensure that your document is well-lit and easily readable. Avoid using any image editing software on your document.
Use Your School Email Address: If possible, apply with your academic email address (e.g., @school.edu). This helps GitHub verify your status more quickly.
Follow Up if Necessary: If you’re notified that your documentation was “manipulated” or “unclear,” consider resubmitting a clearer, unedited image. You can reach out to GitHub Education support for additional help [here](https://support.github.com/contact/education).
Additional Advice
If your application is declined, don’t get discouraged. You can reapply after addressing the specific feedback provided by GitHub, and they’ll review your application again. The GitHub Education team is very responsive and supportive, so don’t hesitate to reach out if you need further clarification.

Good luck! Getting access to the GitHub Student Developer Pack is an excellent step to enhance your learning journey, and I hope it proves valuable for your studies.

**Comment by freya274** [comment]
To qualify for GitHub's Student Developer Pack, you must be enrolled in a degree-granting institution like a high school or university and [Mobcup](Mobcup.bond) the age requirement. For documentation, you’ll need to provide a school-issued email or upload proof of enrollment, similar to platforms that require user verification.

**Comment by rajendrakumaryadav** [comment]
Hello,
I am student of Law, May I apply for Github's Student Developers Pack?
I have my degree in M. Sc (CS) in past.

1. I have Institute ID, which is in Hindi.
2. I have result on University website.

Help me to understand it.

Thanks

**Reply by Chiragmanral** [reply]
just apply for it if you can, there might be a chance you are not eligible for the pack because you are already passed out and this pack is for only students. But just give it a try, let's see what happens

**Comment by DhiyaneshwarSM** [comment]
Its been 5 days since i applied for the same, but the application is pending, can you please check at the earliest

**Reply by Chiragmanral** [reply]
The GitHub Education team is very responsive and supportive, so don’t hesitate to reach out if you need further clarification. Just reapply for it if you can cancel from your side or just raise a issue for tour case.
Thanks
